Ruddarr/Views/Movies/MovieContextMenu.swift

```diff
@@ -11,13 +13,31 @@ struct MovieContextMenu: View {
         if let callsheetUrl = callsheet {
             link(name: "Callsheet", url: callsheetUrl)
         }
+        
+        Divider()
+        
+        Button("Search", systemImage: "magnifyingglass") {
```

Comment:
I think these should match exactly what the buttons say, "Automatic Search" I think.